.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build Binaries
+
+on:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  # Build and packages all the things
+  build-artifacts:
+    strategy:
+      fail-fast: false
+      matrix:
+        # For these target platforms
+        include:
+        - os: ubuntu-20.04
+          deps-script: sudo apt-get install -y protobuf-compiler
+          dist-args: --artifacts=global
+          install-dist: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/axodotdev/cargo-dist/releases/download/v0.0.7/cargo-dist-installer.sh | sh
+        # - os: macos-11
+        #   deps-script: brew install protobuf
+        #   dist-args: --artifacts=local --target=aarch64-apple-darwin --target=x86_64-apple-darwin
+        #   install-dist: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/axodotdev/cargo-dist/releases/download/v0.0.7/cargo-dist-installer.sh | sh
+        - os: ubuntu-20.04
```

Comment:
Is the duplicated `os: ubuntu-20.04` intended?

Reply:
I see one is marked as `--artifacts=global` and other `--artifacts=local --target=x86_64-unknown-linux-gnu`

Reply:
The global one is the one that generates additional release metadata